.gitignore

```diff
@@ -2,6 +2,7 @@
 composer.lock
 vendor/*
 collectors
+conf/params.local.xml
```

Comment:
I don't think that's something you would want. Using that argumentation, You could also replace `composer.lock` by `*.lock` for example..

Anyway, if you really want me to do that, I don't even think your line 6 is needed, since that one is already checked in..

Reply:
Ok, you've done it in ee3b7cb
I'm closing this conversation

Reply:
Woops sorry my mistake, I picked the wrong conversation ! This one is about the gitignore file sorry
Reopening it !

Reply:
No I didn't

Reply:
Reviewed today, please just apply the changed proposed by @piRGoif upper in this thread and it will be accepted (technically).
Functional review is next week.